### PR TITLE
Reduce discretisation artefacts

### LINKED ISSUE #57: normalisation and discretisation
- The phantom used for the normalisation uses 1 sample per voxel, leading to a jaged boundary. Things will work better when using more samples here https://github.com/UCL/STIR-GATE-Connection/blob/dc58ecadbcf20706ebea94846faf77eef7d5a383/ExamplePhantoms/STIRparFiles/D690FOVCylinder.par#L20-L22
- Normalisation factors depend on the projector. We currently use the default projector, which is the ray-tracing matrix with a single LOR. This has its own discretisation errors, which don't really match what GATE does. The estimated efficiencies therefore try to compensate for this, but this means they are not very good for smaller phantoms. Recommended therefore to use [a par file](https://github.com/UCL/STIR/blob/master/examples/samples/forward_projector_proj_matrix_ray_tracing.par) saying `number of rays in tangential direction to trace for each bin:= 5` or similar here https://github.com/UCL/STIR-GATE-Connection/blob/dc58ecadbcf20706ebea94846faf77eef7d5a383/VoxelisedSimulation/SubScripts/EstimateGATESTIRNorm.sh#L54

Note that the doc should make it clear that the norm factors are for the given projector.

### PR DESCRIPTION
**Pull request by KrisThielemans**
- increase number of samples per voxel
- increase number of STIR LORs used

Fixes #57

### CODE REVIEW

**Comment by KrisThielemans** [comment]
@francescaleek have a look at this

File: VoxelisedSimulation/SubScripts/README.md [hunk 1]
```diff
@@ -20,7 +20,8 @@ This directory contains a collection of scripts that are key to automatically fi
 
 Scripts
 =======
-* `EstimateGATESTIRNorm.sh`: Creates normalisation correction sinogram from GATE measured data and STIR model data. 
+* `EstimateGATESTIRNorm.sh`: Creates normalisation correction sinogram from GATE measured data and STIR model data.<br />
+   *Warning:* You will need to make sure that the projector that you use during reconstruction matches the one used for the estimation of the norm factors. See `forward_projector_proj_matrix_ray_tracing.par`.
```
Comment (robbietuk) [inline VoxelisedSimulation/SubScripts/README.md:24]: Does `forward_projector_proj_matrix_ray_tracing.par` exist?

Comment (KrisThielemans) [reply VoxelisedSimulation/SubScripts/README.md:24]: it does now!

**Comment by robbietuk** [comment]
Anything more?

**Comment by KrisThielemans** [comment]
not from me. if it works, merge it!

**Pull request merged**